src/posts/blog/7-budget/budget.ts

```diff
@@ -0,0 +1,90 @@
+import podOverviewChart from './img/pod-overview-chart.png'
+import { ImageMeta } from 'posts/post'
+
+const image: ImageMeta = {
+  url: podOverviewChart,
+  width: 1632,
+  height: 502,
+}
+
+const title =
+  'How our house got our lives back by using a collective “risk budget”'
```

Comment:
Commented in the paper doc as well - since this is primarily to publicize the new Risk Tracker, I think it would be good to work that into the title.

Perhaps "microCOVID Risk Tracker: How our house got our lives back"

You also really don't go into details about what a "risk budget" is, which I would expect if you put the term in quotes in the title.

Reply:
Great suggestion. Done.